<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_widget</class>
 <widget class="QWidget" name="login_widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>138</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="login_label">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="username_label">
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="password_label">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Password:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="username_edit"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="password_edit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="login_button">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>login_button</sender>
   <signal>clicked()</signal>
   <receiver>login_widget</receiver>
   <slot>loginclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginclick()</slot>
 </slots>
</ui>
